src/events/messageDelete.ts

```diff
@@ -19,6 +21,24 @@ export default async function MessageDeleteEvent(discordClient: DiscordClient, m
 		return;
 	}
 
+	await sleep(1000); // wait 1sec for audit logs to generate
+
+	const guild = message.guild;
+	const auditLogs = await guild.fetchAuditLogs({
+		limit: 7,
+		type: 'MESSAGE_DELETE',
+	});
+
+	const auditEntry = auditLogs.entries.find(x => {
```

Comment:
It's quite unclear what is actually happening here and what certain numbers mean. Please add a linked issue, document what the changes are you're making and consider adding comments